src/v2/ion/ui-schema/ion-string-handler.js

```diff
@@ -68,6 +68,13 @@ const createUiSchemaForString = (ionFormField, remediationForm, transformedResp,
     }
   }
 
+  // set optional label for text boxes
+  if(ionFormField.required === false) {
+    if(uiSchema.type === 'text') {
+      uiSchema.sublabel = loc('oie.form.field.optional', 'login');
```

Comment:
Can you please double check that this gets translated properly when a non-English locale is used? I'd assume we're limited by the same constraints as `title`, where this needs to be wrapped in a function.

Reply:
Good point.

Reply:
![Screen Shot 2021-08-24 at 3 28 42 PM](https://user-images.githubusercontent.com/38230587/130698435-8b4bab6c-3c28-4cc4-b537-2c147da9bf8e.png)
 "Optional" is getting translated. does it confirm ?